components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -155,96 +156,116 @@ const ContributorProfileHeader = ({
-                <Button className="hidden sm:inline-flex" variant="primary" href={cardPageUrl(username!)}>
-                  <FaIdCard className="mt-1 mr-1" /> Get Card
-                </Button>
-
-                <Button
-                  onClick={() => handleCopyToClipboard(`${host}/user/${user?.user_metadata.user_name}`)}
-                  className="bg-white sm:hidden"
-                  variant="text"
-                >
-                  <FiCopy className="" />
-                </Button>
-                <Button
-                  onClick={() => handleCopyToClipboard(`${host}/user/${user?.user_metadata.user_name}`)}
-                  className="px-8 py-2 bg-white hidden sm:inline-flex"
-                  variant="text"
-                >
-                  <FiCopy className="mt-1 mr-1" /> Share
-                </Button>
+            <div className="flex justify-center items-center  gap-2 mb-10 md:gap-6 flex-wrap">
+              {user ? (
+                !isOwner && (
+                  <>
+                    {isFollowing ? (
+                      <Button onClick={handleFollowClick} variant="primary" className="group w-[6.25rem] text-center">
+                        <span className="hidden sm:block group-hover:hidden">Following</span>
+                        <span className="block sm:hidden group-hover:block text-center">Unfollow</span>
+                      </Button>
+                    ) : (
+                      <Button variant="primary" className="w-[6.25rem]" onClick={handleFollowClick}>
+                        <HiUserAdd fontSize={20} className="mr-1" /> Follow
+                      </Button>
+                    )}
+                  </>
+                )
+              ) : (
+                <>
+                  <Button
+                    className="sm:hidden"
+                    variant="primary"
+                    onClick={async () =>
+                      handleSignIn({ provider: "github", options: { redirectTo: `${host}/${currentPath}` } })
+                    }
+                  >
+                    <HiUserAdd />
+                  </Button>
+                  <Button
+                    className="w-[6.25rem] hidden sm:inline-flex"
+                    variant="primary"
+                    onClick={async () =>
+                      handleSignIn({ provider: "github", options: { redirectTo: `${host}/${currentPath}` } })
+                    }
+                  >
+                    <HiUserAdd fontSize={20} className="mr-1" /> Follow
+                  </Button>
+                </>
+              )}
 
-                {!isOwner && (
-                  <DropdownMenuTrigger title="More options" className="p-2 mr-3 bg-white rounded-full cursor-pointer ">
-                    <TfiMoreAlt size={20} className="" />
-                  </DropdownMenuTrigger>
-                )}
-              </div>
+              <Button className="sm:hidden" variant="primary" href={cardPageUrl(username!)}>
```

Comment:
```suggestion
              <Button className="sm:hidden" variant="text" href={cardPageUrl(username!)}>
```

Reply:
> Do you want to change the Get Card Button variant to text for mobile?

Reply:
yes... so it is with and the icon is orange

Reply:
Done 👌

Reply:
sorry for the back and forth 😄, can you please add a `bg-white` class name to that button to make it white?

Reply:
sure, done 🚀